packages/examples/react-with-custom-modal/sdk-copy.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Make sure to start from base workspace folder
+reldir="$( dirname -- "$0"; )";
+cd "$reldir/../../..";
+SDK_WORKSPACE_DIR="$( pwd; )";
+COMM_LAYER_DIR="$SDK_WORKSPACE_DIR/packages/sdk-communication-layer"
+SDK_DIR="$SDK_WORKSPACE_DIR/packages/sdk"
+SDK_REACT_DIR="$SDK_WORKSPACE_DIR/packages/sdk-react"
+SDK_REACT_UI_DIR="$SDK_WORKSPACE_DIR/packages/sdk-react-ui"
+SDK_INSTALL_MODAL_WEB_DIR="$SDK_WORKSPACE_DIR/packages/sdk-install-modal-web"
+
+DAPP_DIR="$SDK_WORKSPACE_DIR/packages/examples/create-react-app"
```

Comment:
Should this be?
DAPP_DIR="$SDK_WORKSPACE_DIR/packages/examples/react-with-custom-modal"

Reply:
Changed :)